app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  has_one :workshop
+  belongs_to :sponsor
+  has_many :attends
```

Comment:
is `attends` the right model name? We're talking in the tickets about 'attendees'...